src/Lecture1.hs

```diff
@@ -50,7 +51,8 @@ Explanation: @sumOfSquares 3 4@ should be equal to @9 + 16@ and this
 is 25.
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-sumOfSquares x y = error "TODO!"
+sumOfSquares :: Integer -> Integer -> Integer
+sumOfSquares x y = x * x + y * y
```

Comment:
Absolutely right! 👍🏼 
You can also use a standard function [`(^)`](https://hackage.haskell.org/package/base-4.16.0.0/docs/Prelude.html#v:-94-) for finding the square of a number, e.g. `x ^ 2` 🙂